### PR TITLE
samples: BLE: changed default Settings backend to NVS

### PR DESCRIPTION
**Pull request by grochu**
Changed the default backend for Settings from FCB to NVS.

As per NCSDK-3291

### CODE REVIEW

**Comment by grochu** [comment]
@joerchan, @kapi-no could you take a look?

File: samples/bluetooth/central_bas/prj.conf [hunk 1]
```diff
@@ -20,8 +20,8 @@ CONFIG_BT_SETTINGS=y
 CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
-CONFIG_FCB=y
+CONFIG_NVS=y
```
Comment (kapi-no) [inline samples/bluetooth/central_bas/prj.conf]: This line is no longer necessary. I think that CONFIG_SETTINGS_NVS/FCB activates CONFIG_NVS/FCB option.
See the upstream sample for reference:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/bluetooth/peripheral/prj.conf

And it's better to remove redundant configuration :)

Comment (grochu) [reply samples/bluetooth/central_bas/prj.conf]: Changed as in https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/bluetooth/peripheral/prj.conf

**Comment by joerchan** [comment]
@grochu Please update commit message to use Bluetooth instead of BLE.

**Pull request merged**